src/components/SocialNetworksList/SocialNetworksList.react.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+
+import * as React from "react";
+import cn from "classnames";
+import { List, Icon, Button } from "../";
+
+type itemObject = {|
+  name: string,
+  label?: string,
+  to?: string,
+  tooltip?: string,
+  color?: string,
+|};
+
+type Props = {|
+  +children?: React.Node,
+  +className?: string,
+  +asButtons?: boolean,
+  +prefix?: string,
+  +itemsObjects?: Array<itemObject>,
+  +items?: Array<React.Node>,
+|};
+
+function listItemFromObjectFactory(asButtons, iconPrefix) {
```

Comment:
Here we should type the two arguments as well as the return type for this function